### ISSUE TITLE
Make icon follow font-size, color ......

### ISSUE THREAD

**Issue by aron19951209** [feathericons/feather#727]
Making icon follow font-size and color would be useful.

like font awesome：https://fontawesome.com/

**Comment by locness3** [comment]
Sadly, this would be hard since feather is using SVGs, which are sized with a width and height. Font awesome icons are special characters in their font.

**Comment by CreativeBulma** [comment]
Wrong. since v5 fontawesome is using SVG by default. And its following font-size. Which is as @aron19951209 says very useful.

**Comment by colebemis** [comment]
You can size the icons relative to the font size using `em` units (e.g. `1em`, `1.5em`): https://www.impressivewebs.com/understanding-em-units-css/

**Comment by CreativeBulma** [comment]
Currently this behavior is not working due to the fact that feather javascript part force width and size to the SVG. The current solution is to set <code>width</code> and <code>height</code> attribute on the html tag to tell feather to set a specific size to the icon. But if we just change the font size feather's icons stay sticked to the width/height and not the font size.

**Comment by colebemis** [comment]
Here's an example of how it can be done: https://codepen.io/team/GitHub/pen/pooqowJ

**Comment by CreativeBulma** [comment]
@colebemis my bad I didn't undestand you meant setting <code>em</code> unit on feather initialization.
Thanks a lot for the help.

**Comment by locness3** [comment]
> Wrong. since v5 fontawesome is using SVG by default.

Yeah, but not inline SVGs ? AFAIK it inserts icons with `:before` element, it's CSS only.. I could be wrong though.

**Comment by CreativeBulma** [comment]
> > Wrong. since v5 fontawesome is using SVG by default.
> 
> Yeah, but not inline SVGs ? AFAIK it inserts icons with `:before` element, it's CSS only.. I could be wrong though.

Yes it does replace 
```html
<i class="fas fa-..."></i>
```
with inline SVG.

You just have to include JavaScript file generated from https://fontawesome.com/start

**Comment by locness3** [comment]
Oh OK.

**Comment by hamlet** [comment]
> Here's an example of how it can be done: https://codepen.io/team/GitHub/pen/pooqowJ

This should be on the front page :
`feather.replace({width: '1em', height: '1em'})`

**Comment by reresetyawan** [comment]
In React, i used this way:
`<Icon.Search size={16}  />`

and it's work!

**Comment by hairullana** [comment]
> You can size the icons relative to the font size using `em` units (e.g. `1em`, `1.5em`): https://www.impressivewebs.com/understanding-em-units-css/

I still can't change the font size with em units
![image](https://user-images.githubusercontent.com/56705867/215940523-df25b894-b94b-4579-8c83-712e261b6333.png)

The icons follow that attribute to make sure the size
![image](https://user-images.githubusercontent.com/56705867/215940468-38fafada-5020-41dd-a6b4-66f662f3c192.png)

**Comment by hairullana** [comment]
> > Here's an example of how it can be done: https://codepen.io/team/GitHub/pen/pooqowJ
> 
> This should be on the front page : `feather.replace({width: '1em', height: '1em'})`

But it'll change all icon
How to change some icon, because sometime I need icon with big size, sometime with small size?

**Comment by hairullana** [comment]
The icon will render as an asset, not fonts, so you can't use font-size to change the icon's size.
![image](https://user-images.githubusercontent.com/56705867/215941946-15cd1db7-701f-4181-82d4-bb0ca8dfcc19.png)

The only way to change size of feather icons is set width and height attribute.
![image](https://user-images.githubusercontent.com/56705867/215941744-48411b6f-ff8e-4ede-91c8-462559b4dc2e.png)

This is the result
![image](https://user-images.githubusercontent.com/56705867/215941999-88a2df27-e690-46c4-815c-2685d1e53185.png)
